src/aurora/erfwriter.cpp

```diff
@@ -113,12 +116,63 @@ ERFWriter::ERFWriter(uint32 id, uint32 fileCount, Common::SeekableWriteStream &s
 			break;
 		}
 
+		case kERFVersion22: {
```

Comment:
The constructor and also the method `add()` are getting a bit long. Maybe it's time to split the version bits into individual private functions, like the ERF reader does? I.e. `initV10()`/`initV20()`/`initV22()` and `addV10()`/`addV20()`/`addV22()`